src/Framework/MSBuildEventSource.cs

```diff
@@ -8,11 +8,16 @@ namespace Microsoft.Build.Eventing
     /// This captures information of how various key methods of building with MSBuild ran.
     /// </summary>
     /// <remarks>
-    /// Changes to existing event method signatures will not be reflected unless you update the <see cref="EventAttribute.Version" /> property or assign a new event ID.
+    /// Changes to existing event method signatures will not be reflected unless you update the <see cref="MSBuildEventSource.EventVersion" /> constant or assign a new event ID.
     /// </remarks>
     [EventSource(Name = "Microsoft-Build")]
     internal sealed class MSBuildEventSource : EventSource
     {
+        /// <summary>
+        /// Represents the version of the events.  Be sure to increment this any time you modify an existing event method.
+        /// </summary>
+        public const int EventVersion = 100;
```

Comment:
Is there some reason you starting at 100 instead of 1? Does the version comparer think 9 > 10?

Reply:
Ping 😄